Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -1378,6 +1378,21 @@ void setRGPProps(const std::string_view symb, Atom *res) {
   std::string symbc(symb);
   res->setProp(common_properties::dummyLabel, symbc);
 }
+
+void lookupAtomicNumber(Atom *res, const std::string &symb, bool strictParsing) {
+  try {
```

Comment:
It seems like using exceptions is a pretty heavy hammer, is there another way to tell whether we have a valid atomic number?

Reply:
I take it back, it looks like this should be (TM) faster than an if statement.